packages/platforms/delta/x86-64/x86-64-delta-ag7648/platform-config/r0/src/python/x86_64_delta_ag7648_r0/__init__.py

```diff
@@ -0,0 +1,31 @@
+from onl.platform.base import *
+from onl.platform.delta import *
+
+class OnlPlatform_x86_64_delta_ag7648_r0(OnlPlatformDelta):
```

Comment:
modify the file as the require in the new commit 6ed1fae68c7d0f2fc4238edd76ba12cc11e68df9